py-polars/polars/dataframe/frame.py

```diff
@@ -6095,6 +6095,7 @@ def pivot(
         └─────┴─────┴─────┴─────┘
 
         Run an expression as aggregation function
+        
```

Comment:
you just need to remove the whitespace, i.e.
```suggestion

```

Reply:
Thanks!